trade-finance-manager-api/src/v1/content-strings/index.js

```diff
@@ -379,7 +379,7 @@ const DEAL = {
             'human cloning',
             'pornography',
             'tobacco or gambling',
-            'coal or coal power related',
+            'coal, oil, gas or fossil fuel energy',
```

Comment:
LGTM, but should this be a new version i.e `2.1`? 

Or if it's missing in this version, are there data updates and notifications to end users required, for deals that have already been submitted?

Reply:
Yes we did miss it in `2.0`.

Reply:
OK - RE previously submitted deals and changing the EC, I think this OK - just noticed this is for TFM, so internal only. I was thinking about the user facing EC's, didn't spot it was for TFM only